### DISCUSSION TITLE
vue-cli4 Webpack配置问题

### DISCUSSION THREAD

**Discussion by Z-Mad** [twbs/bootstrap#31688] [General] [answered]
老版本需要安装依赖sass-loader，postcss-loader和Autoprefixer
{
  test: /\.(scss)$/,
  use: [{
    loader: 'style-loader', // inject CSS to page
  }, {
    loader: 'css-loader', // translates CSS into CommonJS modules
  }, {
    loader: 'postcss-loader', // Run postcss actions
    options: {
      plugins: function () { // postcss plugins, can be exported to postcss.config.js
        return [
          require('autoprefixer')
        ];
      }
    }
  }, {
    loader: 'sass-loader' // compiles Sass to CSS
  }]
},
但是在vue-cli4应该怎么配置？

**Comment by Johann-S** [comment, marked as answer]
Sorry, but we cannot help with how-to or general troubleshooting issues here; only bug reports and feature requests. You'll want to ask around in the official Slack team or Stack Overflow. [See our readme for more details](https://github.com/twbs/bootstrap#community).
